int32[] motor_position